index.bs

```diff
@@ -3605,169 +3605,164 @@ the following algorithm returns <i>true</i>.
     </dl>
 
     <table id="distinguishable-table" class="matrix data complex">
+        <thead>
+            <tr>
+                <th class="corner"></th>
+                <th><div><span>undefined</span></div>
+                <th><div><span>boolean</span></div>
+                <th><div><span>numeric types</span></div>
+                <th><div><span>bigint</span></div>
+                <th><div><span>string types</span></div>
+                <th><div><span>object</span></div>
+                <th><div><span>symbol</span></div>
+                <th><div><span>interface-like</span></div>
+                <th><div><span>callback function</span></div>
+                <th><div><span>dictionary-like</span></div>
+                <th><div><span>sequence-like</span></div>
+        </thead>
```

Comment:
Note that every other table in this specification includes the closing `</t[d|h|r]>` tags.

Reply:
I mean, I'm happy to remove those from all the other tables too, it just wasn't relevant for this PR. I can do it in a followup.

Reply:
Well, doing it in a separate PR would make this diff cleaner.

Reply:
Just to close the loop: are you saying that removing the reformatting of this table is a requested change for an R+ here? Or is it just a drive-by preference, and a commitment to filing a clean-up PR afterwards reformatting the rest of the tables accordingly would be sufficient?

Reply:
It’s mainly a drive‑by preference.

Reply:
In that case, can you r+? Or else tell me that you won't r+ until I make the change?

Reply:
There might be some confusion here; @ExE-Boss is not an editor for this spec and is just doing a drive-by review.